lib/pavilion/resolver.py

```diff
@@ -354,7 +398,33 @@ def load(self, tests: List[str], host: str = None,
         if host is None:
             host = self.base_var_man['sys.sys_name']
 
-        raw_tests = self.load_raw_configs(tests, host, modes)
+        filters = []
+        trimmed_tests = []
+
+        for test in tests:
+            scalar_str = ''
+            scalar_int = 1
+            if fnmatch(test, "*[!.-][*]?*"):
+                test, scalar_str = test.rsplit("*", 1)
+
+                try:
+                    scalar_int = int(scalar_str)
+                except ValueError as err:
+                    raise TestConfigError("Invalid repeat notation".format(err))
```

Comment:
I know I said `sometest*8` was a valid way to ask for tests, but that implies that it's already implemented somewhere in the code. If you can't find where that implementation is, then ask for help. 
_DON'T_ try to re-implement it or handle it yourself without understanding how it works. 

In this case, I was wrong - We used to support that, but I forgot we changed how we did it and it's not really an option anymore. So this whole chunk should go away.

Reply:
Ok, I'm wrong again. I finally found where the multiplication takes place, and it's in `load_raw_configs`.  
Which is where you should be handling all this too.